docs/kb/integration/defect-dojo-integration.md

```diff
@@ -0,0 +1,114 @@
+---
+tags:
+  - DefectDojo
+  - Integration
+description: How to connect Semgrep and DefectDojo
+---
+
+# How to connect Semgrep and DefectDojo
+
+[DefectDojo](https://www.defectdojo.com/) is a well-known vulnerability management tool. It allows you to gather security issues from tools like Semgrep and more. By integrating Semgrep findings into DefectDojo, security teams gain additional insight into the security posture of their systems.
+
+## Starting with integration
+Integrating Semgrep with DefectDojo means dumping findings detected by Semgrep into DefectDojo.
+There are several steps to follow:
+
+1. In DefectDojo:
+    1 Create your **product**. A [product](https://defectdojo-dev.readthedocs.io/en/latest/features.html#products) is DefectDojo's term for a project.
+    2. In that DefectDojo product, create an **engagement** called `semgrep`. An engagement is a channel to import results.
+2. Run a semgrep scan with flags `--json --output report.json` to generate a JSON report.
+
+Now, there are all the necessary elements to start playing with [DefectDojo API](https://documentation.defectdojo.com/integrations/api-v2-docs/).
+
+### DefectDojo API example 
+
+To run API DefectDojo operations (GET, POST, DELETE, etc..), an API token is necessary. To get it, follow [API guide](https://documentation.defectdojo.com/integrations/api-v2-docs/).
+
+* Once got it in your system, declare the environment variable `DEFECT_DOJO_API_TOKEN`
```

Comment:
What system are we using this in? Is this intended to run in a CI environment or local development machine?